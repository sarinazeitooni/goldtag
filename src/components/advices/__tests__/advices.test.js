import React from 'react';
import { shallow } from 'enzyme';
import Advices from "../advices";
describe("advices component" , ()=>{
    it("it should render a div",()=>{
        const container = shallow(<Advices/>)
        expect(container).toMatchSnapshot();
    })
});