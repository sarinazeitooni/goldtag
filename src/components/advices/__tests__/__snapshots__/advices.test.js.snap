// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`advices component it should render a div 1`] = `ShallowWrapper {}`;
